src/test/groovy/org/prebid/server/functional/model/config/AccountDsaConfig.groovy

```diff
@@ -0,0 +1,17 @@
+package org.prebid.server.functional.model.config
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.PropertyNamingStrategies
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import groovy.transform.ToString
+import org.prebid.server.functional.model.request.auction.Dsa
+
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy)
```

Comment:
I suggest removing class-annotation and use the JsonProperty on the particular field

Reply:
Since we’re already utilizing `@JsonNaming` consistently, removing it for a single field might not necessarily simplify matters. So, I vote that it should remain as is.

However, please remove the empty space between `defaultDsa` and `gdprOnly` so that it remains consistent with other model classes

Reply:
@AntoxaAntoxic Historically, we had a convention of using a class-level annotation and only utilized individual field-level annotations if the field falls out of the convention. Although this convention wasn't strictly enforced, it still allows for fewer errors due to people forgetting to mark a field with annotation. So I'd like to leave it be.

Reply:
how often is the kebab case used for the fields? I mean for me the `gdprOnly` is an exception field here, and using naming strategy just for a single field doesn't sound reasonable, and since it's only two fields with opposite cases I suggest not having the class annotation at all until we have the third field to judge.

Reply:
In the case of account config, almost entire account config is kebab-case. So using the kebab-case here is not an exception.

Another thing is to be consistent with how this is done in the model package. And it's by default was always done by using class-level annotation and only marking individual fields if they have exceptions. I also don't like using individual field annotation when there is only one field because this then gets copy-pasted too easily.